### PR TITLE
Make expando show correct playback rate at load

### LINKED ISSUE #3872: Playback Rate not updated at expando load
- My browser (copy from https://www.whatismybrowser.com): Chrome 55 on Mac OS X (El Capitan)
- My RES version: 5.3.1
- Am I in reddit beta (yes/no): yes

*What's the problem?*
Expando of a gfycat w/ a non 1.00 playback rate still shows a playback rate of 1.00

https://www.reddit.com/r/test/comments/5mx5te/test/dc8dnje/

### PR DESCRIPTION
**Pull request by WilliamChiu**
Reformats the playback rate option and uses it in the template, closes #3872

### CODE REVIEW

File: lib/modules/showImages.js [hunk 1]
```diff
@@ -1846,6 +1846,9 @@ function videoAdvanced(options) {
 		time,
 	} = options;
 
+	//Format playbackRate for template
+	options.playbackRate = `${options.playbackRate.toFixed(2).replace('.', '.<wbr>')}x`;
```
Comment (andytuba) [inline lib/modules/showImages.js]: Why add `<wbr>` there?

**Comment by erikdesjardins** [comment]
Please change the PR description to "{fixes,resolves,closes} #ISSUE" so the [issue will be autoclosed](https://github.com/blog/1506-closing-issues-via-pull-requests)

File: lib/modules/showImages.js (hunk 1 above)
Comment (WilliamChiu) [inline lib/modules/showImages.js]: alright, good idea

**Comment by WilliamChiu** [comment]
Ah..

**Comment by erikdesjardins** [comment]
Thanks!

**Pull request merged**